src/section.js

```diff
@@ -81,23 +81,14 @@ class Section {
 	 * @param  {method} _request a request method to use for loading
 	 * @return {string} output a serialized XML Document
 	 */
-	render(_request){
+	render(){
 		var rendering = new defer();
 		var rendered = rendering.promise;
 		this.output; // TODO: better way to return this from hooks?
 
-		this.load(_request).
+		this.load().
 			then(function(contents){
-				var serializer;
```

Comment:
There may have been changes to the xml from content hooks, like adding the base tag, so we need to serializer it here. Please add this back in.